examples/sdk-examples/src/components/login/LoginForm.tsx

```diff
@@ -1,169 +0,0 @@
-// (C) 2019-2022 GoodData Corporation
-import React from "react";
-import { withFormik, FormikProps } from "formik";
```

Comment:
We might be able to get rid of formik and yup in the examples (and maybe the whole repo?.

Reply:
Removed